src/content/en/fundamentals/_redirects.yaml

```diff
@@ -163,3 +163,7 @@ redirects:
 
 - from: /web/fundamentals/design-and-ux/input/
   to: /web/fundamentals/design-and-ux/input/forms/
+
+- from: /web/fundamentals/primers/promises
+  to: https://web.dev/promises
```

Comment:
This isn't working in staging but I'm pretty sure it's set up correctly and will work in production

Reply:
It won't work in staging or production IF the original content is still exists. 

For example if `src/content/en/fundamentals/primers/promises.md` exists, the redirect will not work.